lib/nimble_template/helpers/generator.ex

```diff
@@ -24,7 +24,7 @@ defmodule NimbleTemplate.Generator do
 
   def rename_file(old_path, new_path), do: File.rename(old_path, new_path)
 
-  def replace_content(file_path, anchor, content) do
+  def replace_content(file_path, anchor, content, raise_exception \\ true) do
```

Comment:
Shouldn't we instead define another bang function? It is the same for [Enum.fetch/2](https://hexdocs.pm/elixir/Enum.html#fetch/2) and [Enum.fetch!/2](https://hexdocs.pm/elixir/Enum.html#fetch!/2)

```suggestion
  def replace_content(file_path, anchor, content) # existing function
  
  def replace_content!(file_path, anchor, content) # raises an exception
```

This So post is [a good recap](https://stackoverflow.com/questions/32449423/whats-the-meaning-of-and-syntax-in-elixir/32456168#32456168). It is the same convention in Ruby 🤗

Reply:
True, that makes sense, I forgot that convention will make that change :)

Reply:
Actually, we would need to change for many places because the `existing function` raises the exception :D Will open it on a separate PR :D

Reply:
I will create a separate issue for this FYI @olivierobert

Reply:
Issue created https://github.com/nimblehq/elixir-templates/issues/230

Reply:
Changed the name in https://github.com/nimblehq/elixir-templates/pull/288 👌